pipeline_dp/accumulator.py

```diff
@@ -0,0 +1,117 @@
+import abc
+import typing
+import pickle
+from functools import reduce
+
+
+def merge(accumulators: typing.Iterable[
+  'Accumulator'] = None) -> 'Accumulator':
+  """Merges the accumulators.
+
+  Args:
+    accumulators:
+
+  Returns: Accumulator instance with merged values.
+  """
+  unique_accumulator_types = {type(accumulator).__name__ for accumulator in
```

Comment:
Let's assume that `add_accumulator` checks types:
1. The correctness of `add_accumulator` depends on whether types the same and 
2. `add_accumulator`  might be called without `merge`

Nit: remove types checks

Reply:
Done